### PR TITLE
Issue #121: Refactors

### LINKED ISSUE #121: Move to helpers
Probably, things like dds_registration/money.py should be situated in a helpers folder?

### PR DESCRIPTION
**Pull request by lilliputten**
- Issue #121: Refactors: reorganized some modules. Refactors: minor linter fixes. Refactors: sorted out unused templates, re-organized used ones (needs to be checked). Limited produced html prettifying only for prod mode. Synced version number (to the most recent 1.0.3).

- Issue #129: Fixed broken `Payment.data` access approach (restored conversion of a json string to a dict: `self.data["*"]` has replaced by usage of pre-converted data object).

### CODE REVIEW

File: dds_registration/models.py [hunk 1]
```diff
@@ -250,30 +253,46 @@ def invoice_no(self):
...
     def has_unpaid_invoice(self):
-        return self.data["method"] == "INVOICE" and self.status != "PAID"
+        data = self.get_data()
+        return data["method"] == "INVOICE" and self.status != "PAID"
 
     def items(self):
         """Adapt items format for events and membership"""
         pass
 
+    def get_data(self):
```
Comment (cmutel) [inline dds_registration/models.py]: This isn't needed - JSON deserialization is done automatically by Django. All the calls to `.get_data` can be removed.

**Review by cmutel — CHANGES_REQUESTED** [review]
Please delete the `get_data` method and remove its use.

File: dds_registration/models.py (hunk 1 above)
Comment (lilliputten) [reply dds_registration/models.py]: No, that's wrong. This works in templates, but in the code I was getting exceptions about wrong indices (it's expected integer ones for strings).

I was wondering, how could you pass these things into the working code, actually (I've got those in my Saturday fixes also). M.b., it works differently depending on the platform, like dates?(UP: It's not highly likely, so, I don't understand that.)

Comment (lilliputten) [reply dds_registration/models.py]: For example, a traceback screenshot for `{{ payment.payment_label }}` in `profile`:



But I see smth strange: I remember that I could access those json values from templates, now I see that `{{ payment.data.method }}` is silently failing (resulting in an empty string).

Comment (lilliputten) [reply dds_registration/models.py]: Copying my answer from the messenger:

The reason is that the data property has the `str` type there (actually, it's an instance of `str`). And I don't know another way to retrieve the data, except to convert if to the `dict` type.

My quick research snows (on the current stage) that it's already working in a regular django code (in a view) but doesn't work in a template.

Going further and try to find another solution to solve it.

Comment (lilliputten) [reply dds_registration/models.py]: Ok, I can come up with another solution, but it'll be a bit (much?) weirder:

It's possible to create a template tag (some people advise this approach) to get `jsonfield`' data in templates. But in our case, as we don't only need to access the `method` value, but it's human-readable representation, so we'll have to involve one more trick, to fetch a `METHOD_LABELS` value.

At the moment, I can't see any other solutions (I think, I've already spent enough time here) and stop working on this. But if you still want to have other workaround, I can, 'course, continue to research (but not sure that this solution could be found).

UP: Another approach is to try to patch django code itself, but I think it'd be really much more worse.

**Pull request merged**